server/app.js

```diff
@@ -45,6 +47,6 @@ app.use(function(err, req, res, next) {
 
 
 connectDB();
-app.listen(3000, ()=>console.log('server started on port 3000'));
+app.listen(3002, ()=>console.log('server started on port 3002'));
```

Comment:
`app.listen` actually already happens before that on the entry point of the run script, so that line shouldn't have been added before:  https://github.com/hatchways/team-fettuccini/blob/feature-auth-integration/server/bin/www
(notice the port is 3001 also)